src/main.rs

```diff
@@ -84,15 +116,33 @@ fn convert_rw_rules(matches: clap::ArgMatches) {
 fn test(matches: clap::ArgMatches) {
     env_logger::init();
 
-    let start = nasrnn::get_nasrnn();
+    let start = match matches.value_of("model") {
+        Some(model_name) => match model_name {
+            "resnet50" => resnet50::get_resnet50(),
+            "testnet" => testnet::get_testnet(),
+            "benchnet" => benchnet::get_benchnet(),
+            "nasrnn" => nasrnn::get_nasrnn(),
+            "resnext50" => resnext50::get_resnext50(),
+            _ => panic!("The model name is not supported"),
+        },
```

Comment:
Flatten the two matches:
```
let start = match matches.value_of("model") {
  Some("resnet50") => resnet50::get_resnet50(),
  Some("testnet") => testnet::get_testnet(),
  ...
  Some(_) => panic!("The model name is not supported"),
  None => ...
};
```

if `matches.value_of("model")` has type `Option<String>` instead of `Option<&str>`, you will need to use `.as_deref()` to convert it.
```

Reply:
Done